IFC4x3/Sections/Shared element data schemas/Schemas/IfcSharedInfrastructureElements/Entities/IfcGeotechTypicalSection/Documentation.md

```diff
@@ -0,0 +1 @@
+Interval along the alignment/building structure with similar ground conditions, as part of the GeotechSynthesis model that represents the connection between the ground model and the building. Includes key-properties like expected distribution of ground types (reference to GeotechUnits) and baseline-definition of expected ground conditions and potential hazards, and may also include key-information on design like excavation measures, distribution of support types etc.
```

Comment:
I propose to replace 'building' by 'built structure' (2 occurences).

Reply:
Agreed.

Reply:
Done.